src/analytics/analytics-tracker-window.ts

```diff
@@ -1,9 +1,60 @@
+import { isFunction } from 'lodash';
+
 export interface AnalyticsTracker {
     track(step: string, data: any): void;
-    hit(hitName: string, data: any): void;
-    hasPayloadLimit(data: any): boolean;
 }
 
 export default interface AnalyticsTrackerWindow extends Window {
     analytics: AnalyticsTracker;
+    ga(command: string, eventName: string, payload: any): void;
 }
+
+export const sendGoogleAnalytics = (type: string, payload: any) => {
+    const _window = window as unknown as AnalyticsTrackerWindow;
```

Comment:
Instead of trying to cast it to the type that you want, write a type guard and do a type check instead. e.g.:

```ts
function isAnalyticsTrackerWindow(window: window | AnalyticsTrackerWindow): window is AnalyticsTrackerWindow {
    return 'ga' in window && typeof window.ga === 'function';
}

if (isAnalyticsTrackerWindow(window)) {
    window.ga('send', type, {
        // ...
    });
}
```

Reply:
@davidchin done!